tests/disaster-recovery/sc_arbiter/test_shutdowns.py

```diff
@@ -0,0 +1,487 @@
+        sc_obj.get_logwriter_reader_pods(label=constants.LOGREADER_CEPHFS_LABEL)
+        sc_obj.get_logwriter_reader_pods(
+            label=constants.LOGWRITER_RBD_LABEL, exp_num_replicas=2
+        )
+
+        zones = constants.ZONES_LABELS
+        zones.remove("arbiter")
+
+        start_time = datetime.now(timezone.utc)
+        for i in range(iteration):
+            log.info(f"------ Iteration {i+1} ------")
+            if not immediate:
+                start_time = datetime.now(timezone.utc)
+
+            # note the file names created
+            sc_obj.get_logfile_map(label=constants.LOGWRITER_CEPHFS_LABEL)
+            sc_obj.get_logfile_map(label=constants.LOGWRITER_RBD_LABEL)
+
+            # Fetch the nodes in zone that needs to be crashed
+            zone = random.choice(zones)
+            nodes_to_shutdown = sc_obj.get_nodes_in_zone(zone)
+
+            assert (
+                len(nodes_to_shutdown) != 0
+            ), f"There are 0 zone nodes labeled as topology.kubernetes.io/zone={zone}!!"
+
+            try:
+                nodes.stop_nodes(nodes=nodes_to_shutdown)
+                wait_for_nodes_status(
+                    node_names=[node.name for node in nodes_to_shutdown],
+                    status=constants.NODE_NOT_READY,
+                    timeout=300,
+                )
+            except TimeoutExpiredError:
+                log.error("Seems like some nodes didnt shutdown properly!")
+
+            # check ceph accessibility while the nodes are down
+            assert sc_obj.check_ceph_accessibility(
+                timeout=sc_obj.default_shutdown_durarion
+            ), "Something went wrong. not expected. please check rook-ceph logs"
+
+            # start the nodes
+            try:
+                nodes.start_nodes(nodes=nodes_to_shutdown)
+            except Exception:
+                log.error("Something went wrong!")
+
+            # Validate all nodes are in READY state and up
+            retry(
+                (
+                    CommandFailed,
+                    TimeoutError,
+                    AssertionError,
+                    ResourceWrongStatusException,
+                ),
+                tries=30,
+                delay=15,
+            )(wait_for_nodes_status(timeout=1800))
+
+            sc_obj.get_logwriter_reader_pods(label=constants.LOGWRITER_CEPHFS_LABEL)
+            sc_obj.get_logwriter_reader_pods(label=constants.LOGREADER_CEPHFS_LABEL)
+            sc_obj.get_logwriter_reader_pods(
+                label=constants.LOGWRITER_RBD_LABEL, exp_num_replicas=2
+            )
+
+            if not immediate:
+                end_time = datetime.now(timezone.utc)
+                sc_obj.post_failure_checks(
+                    start_time, end_time, wait_for_read_completion=False
+                )
+
+            # TODO: Read pause and Write pause is only expected in the pods that are impacted by the failure
```

Comment:
do we have some github issue to track this so that we dont miss it implementing later?

Reply:
sorry this is already implemented to some extent. but I will make my analysis and if anything is pending I will create a github issue for the same